XM_Dinner/201_Unown_R.yaml

```diff
@@ -0,0 +1,22 @@
+species: Unown

```

Comment:
definitely weak, needs a significant buff. I'd have at least max HP EVs, adamant nature, X-scissor as a main attack over slash (also, move order) and a potentially useful herb like mental. and it might still want more than that